CHANGELOG.md

```diff
@@ -1,18 +1,21 @@
-# 4.1.0
+### Upcoming
+* Split the code into seperate files.
```

Comment:
I agree.  I have just gone looking to find this line also as using 4.1 has broken the project I was working on :/
This should have been a 5.0.0 release